tests/csm/rest/test_capacity_quota.py

```diff
@@ -179,12 +175,12 @@ def test_40632(self):
         assert result, "Verification for get set user failed."
         self.log.info("Response : %s", resp)
         self.log.info("Step 3: Perform put object of max size")
-        self.csm_obj.verify_max_size(max_size)
+        self.csm_obj.verify_max_size(max_size, self.akey, self.skey)
         self.log.info("Step 4: Delete object")
         assert s3_misc.delete_object(
             self.bucket, self.obj_name, self.akey, self.skey), "Failed to delete bucket."
         self.log.info("Step 5: Perform put objects of small size")
-        self.csm_obj.verify_max_objects(max_size, max_objects)
+        self.csm_obj.verify_max_objects(max_size, max_objects, self.akey, self.skey)
```

Comment:
Return value verification missing

Reply:
response for verify_max_object, max_size is verified in the function itself hence it has not been put explicity here .

Reply:
Are u sure that assert in the function works?

Reply:
Tried asserting in an existing lib function, could see it work.

Reply:
resolving